### ISSUE TITLE
Fail to compile due winevulkan_Use_standard_CRT_memory_allocators2

### ISSUE THREAD

**Issue by iWeaker4you** [Frogging-Family/wine-tkg-git#341]
It does not apply the patch winevulkan_Use_standard_CRT_memory_allocators2 correctly

![2021-04-21-08:36:29-screenshot](https://user-images.githubusercontent.com/49424816/115581454-dea60a00-a27c-11eb-83be-0abac5bf661a.png)

**Comment by Fox2Code** [comment]
I got the same bug, but I did some additional research.

This is a hotfix applied only when `_proton_fs_hack="true"`, a temporary workaround is to change it to `_proton_fs_hack="false"` in the config file.

Also, I tried to remove [this line](https://github.com/Frogging-Family/wine-tkg-git/blob/b17f22c541dcb343df5cdd8ce08bd5981c97a1bd/wine-tkg-git/wine-tkg-patches/hotfixes/hotfixer#L502) (Wich adds the failing patch) and the proton fullscreen hax failed, meaning wine changed something in winevulkan that broke these patches.

These are  files I got after doing a test build using `chaotic-staging.cfg` and unmodified wine-tkg build script:

[prepare.log](https://github.com/Frogging-Family/wine-tkg-git/files/6352421/prepare.log)
 [Rejections.zip](https://github.com/Frogging-Family/wine-tkg-git/files/6352464/Rejections.zip) (I put all `.rej` files from `wine-mirror-git/dlls/winevulkan` in the `.zip`)

Also, this is a screen of the bug when I used `chaotic-staging.cfg`:
![winevulkanfails](https://user-images.githubusercontent.com/42963271/115587804-5cdecc80-a2ce-11eb-9338-35a66b71babe.png)

**TLDR:** Wine changed something in their Vulkan implementation that broke `_proton_fs_hack="true"`

**Comment by scriptis** [comment]
@Fox2Code's fix did not work for me; I managed to build by setting `_staging_version` to [`20303a5`](https://github.com/wine-staging/wine-staging/commit/20303a53ec057d74dd0253a536e5fb0d9e6a550f) from 5 days ago. Anything after this appears broken.

**Comment by Tk-Glitch** [comment]
Fixed with https://github.com/Frogging-Family/wine-tkg-git/commit/dc2aebe741337369ef2cc384e444bba6ec254327
